addons/mail/sfu_server/src/config.js

```diff
@@ -0,0 +1,295 @@
+import os from "node:os";
+
+const FALSY_INPUT = new Set(["disable", "false", "none", "no", "0"]);
+
+// ------------------------------------------------------------
+// ------------------   ENV VARIABLES   -----------------------
+// ------------------------------------------------------------
+
+/**
+ * This env variable is <<REQUIRED>>, the base64 encoded key used
+ * for HMAC/SHA256 signing/verification of the JWTs used for authentication.
+ * e.g: AUTH_KEY=u6bsUQEWrHdKIuYplirRnbBmLbrKV5PxKG7DtA71mng=
+ *
+ * @type {string}
+ */
+export const AUTH_KEY = process.env.AUTH_KEY;
+if (!AUTH_KEY && !process.env.JEST_WORKER_ID) {
+    throw new Error(
+        "AUTH_KEY env variable is required, it is not possible to authenticate requests without it"
+    );
+}
+/**
+ * This env variable is <<REQUIRED>>, the server needs to communicate its public IP to the clients as this is the IP
+ * address that will be used for RTC connections. It can the local IP when testing locally.
+ * e.g: PUBLIC_IP=190.165.1.70
+ *
+ * @type {string}
+ */
+export const PUBLIC_IP = process.env.PUBLIC_IP;
+if (!PUBLIC_IP && !process.env.JEST_WORKER_ID) {
+    throw new Error(
+        "PUBLIC_IP env variable is required, clients cannot establish webRTC connections without it"
+    );
+}
+/**
+ * The RTC listening IP
+ * e.g: RTC_IP=0.0.0.0
+ *
+ * @type {string}
+ */
+export const RTC_IP = process.env.RTC_IP || "0.0.0.0";
+/**
+ * Whether the server is behind a proxy,
+ * If true, the server will use the headers "x-forwarded-for", "x-forwarded-proto" and "x-forwarded-host".
+ *
+ * e.g: PROXY=1
+ *
+ * @type {boolean}
+ */
+export const PROXY = Boolean(process.env.PROXY);
+/**
+ * The HTTP/WS interface
+ * e.g: IP=localhost
+ *
+ * @type {string}
+ */
+export const HTTP_INTERFACE = process.env.HTTP_INTERFACE || "0.0.0.0";
+/**
+ * Port of HTTP and Websocket, defaults to standard port 80.
+ *
+ * @type {number}
+ */
+export const PORT = Number(process.env.PORT) || 80;
```

Comment:
I would advise against port 80, since only the root user can listen to ports < 1024.